views/js/previewer/plugins/button.tpl

```diff
@@ -0,0 +1,19 @@
+<li
+    data-control="{{control}}"
+    class="small btn-info action{{#if className}} {{className}}{{/if}}"
+    title="{{title}}"
+    role="button"
```

Comment:
Why do we use `role="button"` here and `li` tag? It might have weird bugs in screen readers because in `ul` only direct children allowed is `li` tags.